rpms/SPECS/3.11.0/wazuh-manager-3.11.0.spec

```diff
@@ -0,0 +1,796 @@
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/queue/agentless
+%dir %attr(770, ossec, ossec) %{_localstatedir}/ossec/queue/alerts
+%dir %attr(770, ossec, ossec) %{_localstatedir}/ossec/queue/cluster
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/queue/db
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/queue/diff
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/queue/fts
+%dir %attr(770, ossecr, ossec) %{_localstatedir}/ossec/queue/rids
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/queue/rootcheck
+%dir %attr(770, ossec, ossec) %{_localstatedir}/ossec/queue/ossec
+%dir %attr(760, root, ossec) %{_localstatedir}/ossec/queue/vulnerabilities
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/ruleset
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/ruleset/sca
+%attr(640, root, ossec) %{_localstatedir}/ossec/ruleset/sca/*
+%attr(640, root, ossec) %{_localstatedir}/ossec/ruleset/VERSION
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/ruleset/decoders
+%attr(640, root, ossec) %{_localstatedir}/ossec/ruleset/decoders/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/ruleset/rules
+%attr(640, root, ossec) %{_localstatedir}/ossec/ruleset/rules/*
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/.ssh
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/stats
+%dir %attr(1770, root, ossec) %{_localstatedir}/ossec/tmp
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/var
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/db
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/db/agents
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/download
+%dir %attr(770, ossec, ossec) %{_localstatedir}/ossec/var/multigroups
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/run
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/selinux
+%attr(640, root, ossec) %{_localstatedir}/ossec/var/selinux/*
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/upgrade
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/wodles
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/aws
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/aws/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/azure
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/azure/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/docker
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/docker/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap.*
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/template*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content
+%attr(640, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content/*
+
+%{_initrddir}/*
+
+%changelog
+* Mon May 6 2019 support <info@wazuh.com> - 3.10.0
```

Comment:
The changelog line is missing.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/254/commits/bd9469d9104aaced5b40e1a09a527bbe1cefa707